src/components/footnote-group/footnote-group.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import template from './footnote-group.twig';
+
+<Meta
+  title="Components/Footnote/Footnote Group"
+  argTypes={{
+    compact: { type: { name: 'boolean' } },
+    id_suffix: { type: { name: 'string' } },
+  }}
+/>
+
+# Footnote Group
+
+Used for displaying one or more [Footnotes](/docs/components-footnote-footnote--basic).
+
+There are a couple things that are important to note when using Footnote Groups:
+
+- If you have more then one Footnote Group on a page, you'll need to pass in a `suffix_id` to the Footnote Groups to ensure they use unique IDs.
+- Footnote Groups include a [visually hidden](/docs/utilities-display--hidden-visually) heading. By default it is an `h2` saying "Footnotes". This can be customized with the `heading_tag` and `heading_text` properties.
+
+## Basic usage
+
+<Canvas>
+  <Story name="Basic">
+    {(args) =>
+      template({
+        items: ['Footnote 1', 'Footnote 2', 'Footnote 3', 'Footnote 4'],
+        ...args,
+      })
+    }
+  </Story>
+</Canvas>
+
+## Compact
+
+The Compact attribute can be added to reduce the size and spacing of footnote groups, as well as remove their top border. This can be helpful when styling footnote groups in blog comments.
+
+<Canvas>
+  <Story
+    name="Compact"
+    args={{
+      compact: true,
+    }}
+  >
+    {(args) =>
+      template({
+        items: ['Footnote 1', 'Footnote 2', 'Footnote 3', 'Footnote 4'],
+        ...args,
+      })
+    }
+  </Story>
+</Canvas>
+
+## Template Properties
+
+- `items`: An array of strings
+- `id_suffix` (string) should be used whenever there are multiple sets of footnotes on one page, to ensure the components use unique IDs.
+- `heading_tag` (string) the tag to use for the heading. Defaults to `h1`.
+- `heading_text` (string) the heading text. Defaults to `Footnotes`.
```

Comment:
@Paul-Hebert Can you add a `compact` description here also, thanks! 😉

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/c951e446c177601978bc4dea3cd3a73e7b94541e